libcrux-ml-dsa/src/simd/avx2/ntt.rs

```diff
@@ -275,8 +485,8 @@ unsafe fn ntt_at_layer_7_and_6(re: &mut AVX2RingElement) {
         }};
     }
 
-    const STEP_BY_7: usize = 2 * COEFFICIENTS_IN_SIMD_UNIT;
-    const STEP_BY_6: usize = (1 << 6) / COEFFICIENTS_IN_SIMD_UNIT;
+    const STEP_BY_7: usize = 16; //2 * COEFFICIENTS_IN_SIMD_UNIT;
```

Comment:
The proof can't handle this? Let's add a comment why we can't compute these values.

Reply:
added a note.